common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -483,6 +583,46 @@ public byte getByte(int i) {
     return Platform.getByte(base, offset + i);
   }
 
+  /**
+   * Returns the code point at position `i`.
```

Comment:
we should be more clear about what position means here. It's the position of char, not byte